src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,173 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final String LOGIN_ALREADY_TAKEN = "This login is already taken";
+    private static final String LOGIN_TOO_SHORT = "Login cannot be null or less than 6 characters";
+    private static final String PASSWORD_TOO_SHORT
```

Comment:
There is a missing semicolon at the end of the `PASSWORD_TOO_SHORT` constant declaration. This will cause a syntax error.